domains/google.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "simplykitter",
+    "email": "kyztrx.lol@gmail.com"
+  },
+  "record": {
+    "A": "69.30.249.53"
```

Comment:
change it to the correct format of A record (check here https://www.is-a.dev/docs/domain-structure/)